internal/pkg/config/v1/config.go

```diff
@@ -133,6 +134,25 @@ func New() *Config {
 	}
 }
 
+func (c *Config) DecryptContextStates() error {
+	if c.CurrentContext != "" {
+		context := c.Context()
```

Comment:
This might be preferred?
```suggestion
	if context := c.Context(); context != nil {
```